docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,183 @@
+
+## Get a Kubernetes Cluster
+
+<ul class="nav nav-tabs">
+<li class="active"><a href="#setup-mac-brew" data-toggle="tab">macOS via Homebrew</a></li>
+<li><a href="#setup-mac-linux" data-toggle="tab">macOS / Linux</a></li>
+<li><a href="#setup-mac-linux" data-toggle="tab">Windows</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="setup-mac-brew" markdown="1">
+```
+brew upgrade
+brew install kind
+brew install kubectl
+brew install helm
+
+kind create cluster --image kindest/node:v1.15.11 --wait 5m
+```
+</div>
+<div class="tab-pane fade" id="setup-mac-linux" markdown="1">
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+</div>
+<div class="tab-pane fade" id="setup-windows" markdown="1">
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+</div>
+</div>
+
+## Install Crossplane
+Install the alpha version of Crossplane, which is suitable for community use and testing:
+<ul class="nav nav-tabs">
+<li class="active"><a href="#install-tab-helm3" data-toggle="tab">Helm 3</a></li>
+<li><a href="#install-tab-helm2" data-toggle="tab">Helm 2</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="install-tab-helm3" markdown="1">
+```
+kubectl create namespace crossplane-system
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+</div>
+<div class="tab-pane fade" id="install-tab-helm2" markdown="1">
+```
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+</div>
+</div>
+
+## Install Crossplane CLI
+The Crossplane CLI adds a set of `kubectl crossplane` commands to simplify common tasks:
+```
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane-cli/master/bootstrap.sh | bash
+```
+
+## Select Provider
+In order for Crossplane to be able to manage resources in a specific cloud
+provider, you will need to install a provider and configure it for Crossplane to use.
+<ul class="nav nav-tabs">
+<li class="active"><a href="#provider-tab-gcp" data-toggle="tab">GCP</a></li>
+<li><a href="#provider-tab-aws" data-toggle="tab">AWS</a></li>
+<li><a href="#provider-tab-azure" data-toggle="tab">Azure</a></li>
+<li><a href="#provider-tab-alibaba" data-toggle="tab">Alibaba</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="provider-tab-gcp" markdown="1">
+### Install GCP Provider
+```
+PACKAGE=crossplane/provider-gcp:v0.9.0
```

Comment:
```suggestion
PACKAGE=crossplane/provider-gcp:master
```

Guessing we will probably update the version tag for each provider on the release branch 👍

Reply:
fixed